.travis.yml

```diff
@@ -1,9 +1,9 @@
 language: dart
 dart:
+  - 2.0.0
```

Comment:
why not `stable` here?

Reply:
Want to test on the oldest supported SDK. We've been burned here too many times.